synapse/storage/events.py

```diff
@@ -1817,6 +1817,40 @@ def get_all_new_events_txn(txn):
 
         return self.runInteraction("get_all_new_events", get_all_new_events_txn)
 
+    def _get_state_event_counts_txn(self, txn, room_id):
+        sql = "SELECT COUNT(*)" " FROM state_events" " WHERE room_id=?"
```

Comment:
Why the weird double quoting?

Reply:
this was broken up and then black unbroke it up because it all fit on one line. fixed.